scripts/changelog/gradle_updater.js

```diff
@@ -12,8 +15,9 @@
 const versionModule = require("./version_utility.js");
 
 const versionNameRegex = /(versionName ")(.+)(")/gm;
+const versionCodeRegex = /(versionCode )([0-9]+)/gm;
 
-module.exports.readVersion = function(contents) {
+module.exports.readVersion = function (contents) {
```

Comment:
Okay. According to [this](https://www.crockford.com/code.html), this is correct.

> if a function literal is anonymous, there should be one space between the word function and the